py2store/persisters/sqlserver_w_pyodbc.py

```diff
@@ -0,0 +1,121 @@
+import platform
+import subprocess
+from collections.abc import MutableMapping
+
+
+class SQLServerPersister(MutableMapping):
+    def __init__(self, conn_protocol='tcp', host='localhost', port='1433', db_username='SA', db_pass='Admin123x',
+                 db_name='py2store', table_name='person', primary_key='id', data_fields=('name',)):
+
+        self.__check_dependencies()
+        self._sql_server_client = pyodbc.connect('DRIVER={{ODBC Driver 17 for SQL Server}};'
+                                                 'SERVER={}:{},{};'
+                                                 'DATABASE={};'
+                                                 'UID={};'
+                                                 'PWD={}'
+                                                 .format(conn_protocol, host, port, db_name, db_username, db_pass))
+
+        self._cursor = self._sql_server_client.cursor()
+        self._table_name = table_name
```

Comment:
We need to make sure this table will be created, if it's not present, so user dont have to worry about it.

Reply:
Done https://github.com/i2mint/py2store/pull/30/files#diff-a7ee3ec855fb8f76cd4fc6b530d67f40R56